data/campaigns/Heir_To_The_Throne/scenarios/15_The_Lost_General.cfg

```diff
@@ -347,10 +347,48 @@
             y=1-23
             side=1
         [/filter]
+
+        [sound]
+             name=thunderstick.ogg
+        [/sound]
+        [delay]
+            time=200
+        [/delay]
+
+        # This dialog has several purposes:
+        # 1. Clue to the player that he's exploring in the right direction
+        # 2. Clue to the player to hurry up, in order to save Geldar
+        # 3. An excuse to play the thunderstick sound :)
         [message]
-            speaker=unit
-            message= _ "These passages seem to have been used recently."
+            speaker=Konrad
+            message=_ "<i>(motions for silence)</i> Listen... There’s rather a fray up ahead. But what was that thunderous noise?"
```

Comment:
What do you mean by:

> There's rather a fray up ahead.

I'd actually just have him ask "What was that thunderous noise?", that's probably good enough to get the point across.

Reply:
That sentence is referring to the shrouded battle between dwarves and orcs/trolls. Yes, let's make it shorter. Do you think we can retain the "motions for silence"?

And by the way, I'm not sure I like the repetition of thunderous/thunderstick, but I didn't find a synonym.

Reply:
Oh yeah, you can leave the "motions for silence", I was referring to the spoken text. Deafening, clamorous, raucous, piercing, booming, and cacophonous are synonyms I can think of that might fit. There's others too, but you can probably just choose from those if you like.